Packs/AzureNetworkSecurityGroups/Integrations/AzureNetworkSecurityGroups/AzureNetworkSecurityGroups.yml

```diff
@@ -613,6 +613,296 @@ script:
     - contextPath: AzureNSG.ResourceGroup.tags.type
       description: 'The type tag associated with the Azure Network Security Groups resource group.'
       type: String
+  - description: Gets public IP addresses in a resource group.
+    name: azure-nsg-public-ip-addresses-list
+    outputs:
+    - contextPath: AzureNSG.PublicIPAdress.name
+      description: The public ip address's name.
+      type: String
+    - contextPath: AzureNSG.PublicIPAdress.id
+      description: The public ip address's id.
+      type: String
+    - contextPath: AzureNSG.PublicIPAdress.etag
+      description: The public ip address's etag.
+      type: String
+    - contextPath: AzureNSG.PublicIPAdress.provisioningState
+      description: The public ip address's provisioning state.
+      type: String
+    - contextPath: AzureNSG.PublicIPAdress.publicIPAddressVersion
+      description: The public ip address's version.
+      type: String
+    - contextPath: AzureNSG.PublicIPAdress.ipAddress
+      description: The public ip address's ip address.
+      type: String
+    - contextPath: AzureNSG.PublicIPAdress.domainNameLabel
+      description: The public ip address's domain name label.
+      type: String
+    - contextPath: AzureNSG.PublicIPAdress.fqdn
+      description: The public ip address's fully qualified domain name (FQDN).
+      type: String
+    arguments:
```

Comment:
Put the arguments before the outputs, so it will be clear that this command receives arguments. (In all other commands too)